### PR TITLE
Update SBpipe version.

### PR DESCRIPTION
**Pull request by pdp10**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/sbpipe/meta.yaml [hunk 1]
```diff
@@ -8,11 +8,11 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
-  sha256: 5ae5fe0332ea76567f9f5405abda27f2631146ff6c735ac809127939f61c9fac
+  sha256: c417db041e8990f41bc56101b7a4be7a861136bac67207dec47b83546ca26d50
 
 build:
   noarch: python
-  number: 1
+  number: 2
```
Comment (pvanheus) [inline recipes/sbpipe/meta.yaml]: Why is the build number not being reset to 0? After all this is a new version, right?

Comment (pdp10) [reply recipes/sbpipe/meta.yaml]: Yes, it is a new version of the same package. Sorry, for some reason I thought the number had to be incremental every time there is a PR for the same package.

**Comment by pdp10** [comment]
@pvanheus 
I received a CondaHTTPError when retrieving the curl package. 
Should CircleCI be restarted?

```
CondaHTTPError: HTTP 502 BAD GATEWAY for url <https://conda.anaconda.org/conda-forge/linux-64/curl-7.60.0-0.tar.bz2>
Elapsed: 00:06.287436
CF-RAY: 42d06ab55c4456db-IAD

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
```

**Comment by bgruening** [comment]
Thanks @pdp10!

**Pull request merged**